docs/attributes-registry/telemetry.md

```diff
@@ -1,43 +1,44 @@
 <!--- Hugo front matter used to generate the website version of this page:
 --->
 
-# Telemetry SDK
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/attribute_namespace.md.j2 -->
 
-## Telemetry SDK Attributes
+# Telemetry
 
-<!-- semconv registry.telemetry(omit_requirement_level) -->
-| Attribute  | Type | Description  | Examples  | Stability |
-|---|---|---|---|---|
-| `telemetry.sdk.language` | string | The language of the telemetry SDK. | `cpp` | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |
-| `telemetry.sdk.name` | string | The name of the telemetry SDK as defined above. [1] | `opentelemetry` | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |
-| `telemetry.sdk.version` | string | The version string of the telemetry SDK. | `1.2.3` | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |
-| `telemetry.distro.name` | string | The name of the auto instrumentation agent or distribution, if used. [2] | `parts-unlimited-java` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `telemetry.distro.version` | string | The version string of the auto instrumentation agent or distribution, if used. | `1.2.3` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+## Telemetry Attributes
 
-**[1]:** The OpenTelemetry SDK MUST set the `telemetry.sdk.name` attribute to `opentelemetry`.
+| Attribute                  | Type   | Description                                                                    | Examples               | Stability                                                        |
+| -------------------------- | ------ | ------------------------------------------------------------------------------ | ---------------------- | ---------------------------------------------------------------- |
+| `telemetry.distro.name`    | string | The name of the auto instrumentation agent or distribution, if used. [1]       | `parts-unlimited-java` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `telemetry.distro.version` | string | The version string of the auto instrumentation agent or distribution, if used. | `1.2.3`                | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
Didn't we want to have stable attributes sorted first?

Reply:
Snippets do that.  We can fix weaver to allow this for attribute registry, but honestly I like attribute registry being in alphabetical order.

Reply:
Looking for confirmation this is ok for the registry from all @open-telemetry/specs-semconv-maintainers -  WDYT?

Reply:
I think @pyohannes worked to get it sorted by name then stability. Then we have best of both worlds.

Reply:
All of the *snippets* are generated by stability.

From the purpose of a "registry" - I think just alphabetical makes more sense.

For the snippets, I agree sorting by stability makes sense as that's where you document *how to use/fill out* specific spans / metrics.   The registry is just that - A dictionary lookup of what's available.

Reply:
Ah all good then as is. 👍